README.md

```diff
@@ -269,6 +269,19 @@ jobs:
 4. **Commit and Push Your Workflow**:
    - Once you commit this workflow file to your repository, the action will be triggered based on the defined events (e.g., on push or pull request).
 
+## Usage with Docker
+Build the image with the following and tagging with `latest`:
+```
+docker build -t podcast-rss-generator:latest .
+```
+To spin up a container from the built image that uses a custom config file and writes out to `myfeed.xml`.
+
+```
+docker run -it --rm -v .:/var podcast-rss-generator:latest --input-file /var/my_podcast_config.yaml --output-file /var/myfeed.xml
```

Comment:
`/var` is a bit of non-standardized choice here. Maybe we use `/opt`, or even `/tmp` here?

Also, `-it` shouldn't be required given that none of these arguments require input.